## 78% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 9
Expected to fail tests: 2
Passed tests: 40
```
```
IDs of failed tests: 702733,703086,702782,704615,702731,727229,703133,702894,702783 
```
```
IDs of expected to fail tests: 727232,703495 
```
<h3>Failed tests (9)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/bottom-button-background&PR_ID=20585&APK_NAME=20585.apk&TR_CASE_IDS=702733,703086,702782,704615,702731,727229,703133,702894,702783">Rerun failed tests</a></li><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596194">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `show-profiles` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:240: in test_restore_multiaccount_with_waku_backup_remove_switch
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 zQ3...dWXh5 was not restored as a contact from waku backup!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5aa14fa3eef943cdab9016f52dd80bb1?auth=bfd368b8234463a84a239777b4fc3b19#229">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596162">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td>

```
critical/chats/test_1_1_public_chats.py:469: in test_1_1_chat_text_message_delete_push_disappear
    if not self.chat_2.chat_message_input.is_element_displayed():
../views/base_element.py:221: in is_element_displayed
    return self.wait_for_visibility_of_element(sec, ignored_exceptions=ignored_exceptions)
../views/base_element.py:145: in wait_for_visibility_of_element
    .until(expected_conditions.visibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:152: in _predicate
    return _element_if_visible(driver.find_element(*locator))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/423ee08f3fcb498aaa2ee7def1fcc845?auth=2b035688601a99f6e74a3e1ac37edf78#425">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/948d3d1ad5554dd49722c5a0e2e357d0?auth=b690cd27e5a605cf34d27637d63c6023#342">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596167">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><p><blockquote><div>Device 1: Long press on `ChatElementByText` until expected element is shown </div><div>Device 1: Find `Button` by `xpath`: `//*[@text="Reply"]` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:129: in test_1_1_chat_emoji_send_reply_and_open_link
    self.chat_1.quote_message(emoji_unicode)
../views/chat_view.py:1041: in quote_message
    self.reply_message_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//*[@text="Reply"]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/423ee08f3fcb498aaa2ee7def1fcc845?auth=2b035688601a99f6e74a3e1ac37edf78#147">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/948d3d1ad5554dd49722c5a0e2e357d0?auth=b690cd27e5a605cf34d27637d63c6023#119">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596160">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `xpath`: `//*[@content-desc='pins-count']//android.widget.TextView` </div><div>Device 1: Find `Button` by `xpath`: `//*[@content-desc='pins-count']//android.widget.TextView` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:198: in test_1_1_chat_pin_messages
    chat.pinned_messages_count.wait_for_element_text(text="2",
../views/base_element.py:190: in wait_for_element_text
    self.driver.fail(message if message else "`%s` is not equal to expected `%s` in %s sec" % (
base_test_case.py:178: in fail
    pytest.fail('Device %s: %s' % (self.number, text))
 Device 1: Pinned messages count is not 2 as expected!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/423ee08f3fcb498aaa2ee7def1fcc845?auth=2b035688601a99f6e74a3e1ac37edf78#185">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/948d3d1ad5554dd49722c5a0e2e357d0?auth=b690cd27e5a605cf34d27637d63c6023#126">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596193">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td>

```
critical/chats/test_public_chat_browsing.py:772: in test_community_mark_all_messages_as_read
    home.navigate_back_to_home_view()
../views/base_view.py:425: in navigate_back_to_home_view
    while not self.chat_floating_screen.is_element_disappeared(1) \
../views/base_element.py:231: in is_element_disappeared
    return self.wait_for_invisibility_of_element(sec)
../views/base_element.py:154: in wait_for_invisibility_of_element
    .until(expected_conditions.invisibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:319: in _predicate
    target = driver.find_element(*target)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8e85eb5806dd477abf2ba9338237ae21?auth=f4ac1699c3d99b06e61d325eb579e0f9#433">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a29e4ad611849e7a2f05d7ce2b9c5b5?auth=a3c9f6a4f639711f0ccc36cb3358c582#395">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596205">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><p><blockquote><div># STEP: Sending messages for edit and delete</div><div>Device 2: Sending message 'message to edit' </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:802: in test_community_edit_delete_message_when_offline
    self.channel_2.send_message(message_to_edit)
../views/chat_view.py:982: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:127: in wait_for_element
    .until(expected_conditions.presence_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:82: in _predicate
    return driver.find_element(*locator)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8e85eb5806dd477abf2ba9338237ae21?auth=f4ac1699c3d99b06e61d325eb579e0f9#436">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a29e4ad611849e7a2f05d7ce2b9c5b5?auth=a3c9f6a4f639711f0ccc36cb3358c582#397">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596188">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td>

```
critical/chats/test_public_chat_browsing.py:672: in test_community_contact_block_unblock_offline
    if not channel.chat_message_input.is_element_displayed():
../views/base_element.py:221: in is_element_displayed
    return self.wait_for_visibility_of_element(sec, ignored_exceptions=ignored_exceptions)
../views/base_element.py:145: in wait_for_visibility_of_element
    .until(expected_conditions.visibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:152: in _predicate
    return _element_if_visible(driver.find_element(*locator))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8e85eb5806dd477abf2ba9338237ae21?auth=f4ac1699c3d99b06e61d325eb579e0f9#430">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a29e4ad611849e7a2f05d7ce2b9c5b5?auth=a3c9f6a4f639711f0ccc36cb3358c582#393">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596168">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//*[starts-with(@text,'test message')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']//*[@content-desc='message-status']/android.widget.TextView` </div><div>Device 2: `Text` is `Sent` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:623: in test_1_1_chat_is_shown_message_sent_delivered_from_offline
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message status was not changed to Delivered, it's Sent after back up online!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0d36a93032e4474598d1388a5b2cc30b?auth=a2fcde051e0f5bcc60c9db34036d9d99#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6864c6c808d948c59723ec0ec16de575?auth=8ab8893b7c15522bdf808e7939225825#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596206">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `top-bar` </div><div>Device 2: Find `Button` by `accessibility id`: `top-bar` </div></blockquote></p>

```
critical/test_wallet.py:147: in test_wallet_send_eth
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Sender balance is not updated on Etherscan, it is 0.0009 but expected to be 0.001
E    Can't find the last transaction for the receiver
```

<br/><br/></td></tr></tbody></table></details><h3>Expected to fail tests (2)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596200">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><p><blockquote><div># STEP: Change device time so chat will be unmuted by timer</div><div>Device 2: Long press on `ChatElement` </div></blockquote></p>

```
critical/chats/test_group_chat.py:495: in test_group_chat_mute_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Chat is still muted after timeout 
```

<b>[[Chat is not unmuted after expected time: https://github.com/status-im/status-mobile/issues/19627]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/513672670a6e4403aa0e2efad1b4af57?auth=83d62a2abcbd0d6777476bf32210ca7a#477">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3d064019527d4edb99a775bb50684506?auth=7f19dd367a32123169e0be5daec4e75d#273">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9dc2917b6c1848c49827f1c4f6af9704?auth=b0412a2774e24d762a49f6c6898b2e72#242">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596209">test_wallet_add_remove_watch_only_account</a>, id: 727232 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `EditBox` by `accessibility id`: `add-address-to-watch` </div><div>Device 1: Type `0x8d2413447ff297d30bdc475f6d5cb00254685aae` to `EditBox` </div></blockquote></p>

```
critical/test_wallet.py:237: in test_wallet_add_remove_watch_only_account
    self.wallet_view.add_watch_only_account(address=address_to_watch, account_name=new_account_name)
../views/wallet_view.py:150: in add_watch_only_account
    self.account_has_activity_label.wait_for_visibility_of_element()
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 1: Text by accessibility id:`account-has-activity` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[Missing networks in account address, https://github.com/status-im/status-mobile/issues/20166]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4f0721ec9e6148989a25fa4b2fc1d126?auth=61d956a7f0bb627cea4419ffe3d65f0d#102">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (40)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596190">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5f58b07ba80448f3b4e5b8943f398b1c?auth=91375f33f6a164a7f28ff67fdc02613b#303">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fe1e91d82c72465fbe25d6ce99ecaa48?auth=b41fedda5d1c6d8e3d58a135816b6562#296">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596173">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5f58b07ba80448f3b4e5b8943f398b1c?auth=91375f33f6a164a7f28ff67fdc02613b#184">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fe1e91d82c72465fbe25d6ce99ecaa48?auth=b41fedda5d1c6d8e3d58a135816b6562#167">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596170">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5f58b07ba80448f3b4e5b8943f398b1c?auth=91375f33f6a164a7f28ff67fdc02613b#135">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fe1e91d82c72465fbe25d6ce99ecaa48?auth=b41fedda5d1c6d8e3d58a135816b6562#109">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596181">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5f58b07ba80448f3b4e5b8943f398b1c?auth=91375f33f6a164a7f28ff67fdc02613b#300">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fe1e91d82c72465fbe25d6ce99ecaa48?auth=b41fedda5d1c6d8e3d58a135816b6562#274">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596203">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5f58b07ba80448f3b4e5b8943f398b1c?auth=91375f33f6a164a7f28ff67fdc02613b#496">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fe1e91d82c72465fbe25d6ce99ecaa48?auth=b41fedda5d1c6d8e3d58a135816b6562#476">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596198">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5aa14fa3eef943cdab9016f52dd80bb1?auth=bfd368b8234463a84a239777b4fc3b19#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596182">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5aa14fa3eef943cdab9016f52dd80bb1?auth=bfd368b8234463a84a239777b4fc3b19#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596202">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5aa14fa3eef943cdab9016f52dd80bb1?auth=bfd368b8234463a84a239777b4fc3b19#361">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596163">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5aa14fa3eef943cdab9016f52dd80bb1?auth=bfd368b8234463a84a239777b4fc3b19#76">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596187">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5aa14fa3eef943cdab9016f52dd80bb1?auth=bfd368b8234463a84a239777b4fc3b19#105">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596208">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4f0721ec9e6148989a25fa4b2fc1d126?auth=61d956a7f0bb627cea4419ffe3d65f0d#38">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596192">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ee79a259b73b44ac95a536decaf96d41?auth=acdf36f3812ab4a39d6ca7344868e791#169">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6ee9e76c768343ddb28ebb38ff3fb6b0?auth=98ff7d0f2e1b05a49c53a34568d0e062#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596191">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ee79a259b73b44ac95a536decaf96d41?auth=acdf36f3812ab4a39d6ca7344868e791#128">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6ee9e76c768343ddb28ebb38ff3fb6b0?auth=98ff7d0f2e1b05a49c53a34568d0e062#100">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596171">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/513672670a6e4403aa0e2efad1b4af57?auth=83d62a2abcbd0d6777476bf32210ca7a#156">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3d064019527d4edb99a775bb50684506?auth=7f19dd367a32123169e0be5daec4e75d#78">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9dc2917b6c1848c49827f1c4f6af9704?auth=b0412a2774e24d762a49f6c6898b2e72#78">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596196">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/513672670a6e4403aa0e2efad1b4af57?auth=83d62a2abcbd0d6777476bf32210ca7a#173">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3d064019527d4edb99a775bb50684506?auth=7f19dd367a32123169e0be5daec4e75d#106">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9dc2917b6c1848c49827f1c4f6af9704?auth=b0412a2774e24d762a49f6c6898b2e72#86">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596161">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/513672670a6e4403aa0e2efad1b4af57?auth=83d62a2abcbd0d6777476bf32210ca7a#357">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3d064019527d4edb99a775bb50684506?auth=7f19dd367a32123169e0be5daec4e75d#230">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9dc2917b6c1848c49827f1c4f6af9704?auth=b0412a2774e24d762a49f6c6898b2e72#228">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. test_group_chat_offline_pn</b> (TestRail link is not found)</td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/513672670a6e4403aa0e2efad1b4af57?auth=83d62a2abcbd0d6777476bf32210ca7a#322">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3d064019527d4edb99a775bb50684506?auth=7f19dd367a32123169e0be5daec4e75d#203">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9dc2917b6c1848c49827f1c4f6af9704?auth=b0412a2774e24d762a49f6c6898b2e72#201">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. test_group_chat_send_image_save_and_share</b> (TestRail link is not found)</td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/513672670a6e4403aa0e2efad1b4af57?auth=83d62a2abcbd0d6777476bf32210ca7a#282">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3d064019527d4edb99a775bb50684506?auth=7f19dd367a32123169e0be5daec4e75d#168">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9dc2917b6c1848c49827f1c4f6af9704?auth=b0412a2774e24d762a49f6c6898b2e72#141">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. test_1_1_chat_push_emoji</b> (TestRail link is not found)</td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/423ee08f3fcb498aaa2ee7def1fcc845?auth=2b035688601a99f6e74a3e1ac37edf78#284">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/948d3d1ad5554dd49722c5a0e2e357d0?auth=b690cd27e5a605cf34d27637d63c6023#201">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. test_1_1_chat_message_reaction</b> (TestRail link is not found)</td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/423ee08f3fcb498aaa2ee7def1fcc845?auth=2b035688601a99f6e74a3e1ac37edf78#95">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/948d3d1ad5554dd49722c5a0e2e357d0?auth=b690cd27e5a605cf34d27637d63c6023#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. test_1_1_chat_edit_message</b> (TestRail link is not found)</td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/423ee08f3fcb498aaa2ee7def1fcc845?auth=2b035688601a99f6e74a3e1ac37edf78#333">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/948d3d1ad5554dd49722c5a0e2e357d0?auth=b690cd27e5a605cf34d27637d63c6023#232">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. test_1_1_chat_non_latin_messages_stack_update_profile_photo</b> (TestRail link is not found)</td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/423ee08f3fcb498aaa2ee7def1fcc845?auth=2b035688601a99f6e74a3e1ac37edf78#226">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/948d3d1ad5554dd49722c5a0e2e357d0?auth=b690cd27e5a605cf34d27637d63c6023#151">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. test_1_1_chat_send_image_save_and_share</b> (TestRail link is not found)</td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/423ee08f3fcb498aaa2ee7def1fcc845?auth=2b035688601a99f6e74a3e1ac37edf78#364">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/948d3d1ad5554dd49722c5a0e2e357d0?auth=b690cd27e5a605cf34d27637d63c6023#282">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. test_community_one_image_send_reply</b> (TestRail link is not found)</td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8e85eb5806dd477abf2ba9338237ae21?auth=f4ac1699c3d99b06e61d325eb579e0f9#262">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a29e4ad611849e7a2f05d7ce2b9c5b5?auth=a3c9f6a4f639711f0ccc36cb3358c582#225">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. test_community_several_images_send_reply</b> (TestRail link is not found)</td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8e85eb5806dd477abf2ba9338237ae21?auth=f4ac1699c3d99b06e61d325eb579e0f9#237">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a29e4ad611849e7a2f05d7ce2b9c5b5?auth=a3c9f6a4f639711f0ccc36cb3358c582#164">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. test_community_message_edit</b> (TestRail link is not found)</td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8e85eb5806dd477abf2ba9338237ae21?auth=f4ac1699c3d99b06e61d325eb579e0f9#180">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a29e4ad611849e7a2f05d7ce2b9c5b5?auth=a3c9f6a4f639711f0ccc36cb3358c582#123">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. test_community_message_send_check_timestamps_sender_username</b> (TestRail link is not found)</td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8e85eb5806dd477abf2ba9338237ae21?auth=f4ac1699c3d99b06e61d325eb579e0f9#162">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a29e4ad611849e7a2f05d7ce2b9c5b5?auth=a3c9f6a4f639711f0ccc36cb3358c582#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. test_community_emoji_send_copy_paste_reply</b> (TestRail link is not found)</td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8e85eb5806dd477abf2ba9338237ae21?auth=f4ac1699c3d99b06e61d325eb579e0f9#311">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a29e4ad611849e7a2f05d7ce2b9c5b5?auth=a3c9f6a4f639711f0ccc36cb3358c582#285">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596178">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8e85eb5806dd477abf2ba9338237ae21?auth=f4ac1699c3d99b06e61d325eb579e0f9#400">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a29e4ad611849e7a2f05d7ce2b9c5b5?auth=a3c9f6a4f639711f0ccc36cb3358c582#363">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596180">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8e85eb5806dd477abf2ba9338237ae21?auth=f4ac1699c3d99b06e61d325eb579e0f9#335">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a29e4ad611849e7a2f05d7ce2b9c5b5?auth=a3c9f6a4f639711f0ccc36cb3358c582#302">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596176">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8e85eb5806dd477abf2ba9338237ae21?auth=f4ac1699c3d99b06e61d325eb579e0f9#204">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a29e4ad611849e7a2f05d7ce2b9c5b5?auth=a3c9f6a4f639711f0ccc36cb3358c582#135">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596169">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0d36a93032e4474598d1388a5b2cc30b?auth=a2fcde051e0f5bcc60c9db34036d9d99#174">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6864c6c808d948c59723ec0ec16de575?auth=8ab8893b7c15522bdf808e7939225825#259">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596201">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0d36a93032e4474598d1388a5b2cc30b?auth=a2fcde051e0f5bcc60c9db34036d9d99#110">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6864c6c808d948c59723ec0ec16de575?auth=8ab8893b7c15522bdf808e7939225825#246">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596207">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596183">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a982048c1e12424f873f27d5e89118ba?auth=207b406e31798ccd29f09de3e916bb93#56">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c11f8f933b1e473a984a7f606a9f4cf3?auth=164b31cdf3bd6a5b0fdf9243466bd53d#55">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596166">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a982048c1e12424f873f27d5e89118ba?auth=207b406e31798ccd29f09de3e916bb93#134">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c11f8f933b1e473a984a7f606a9f4cf3?auth=164b31cdf3bd6a5b0fdf9243466bd53d#214">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596184">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a982048c1e12424f873f27d5e89118ba?auth=207b406e31798ccd29f09de3e916bb93#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c11f8f933b1e473a984a7f606a9f4cf3?auth=164b31cdf3bd6a5b0fdf9243466bd53d#121">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596189">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f1de8694ddbe484db99b5d11927ddab1?auth=0f100b6411c17a8c8aa17eceea9e8891#128">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b06d9526b6754fcdb3d00e793b215ab5?auth=2306c1e234341e84ba831aa1f4559f95#98">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596204">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/850318754fdf423880581e4c683034fc?auth=0127bb0028d08a9e91508a456a31d6e6#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3596165">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/850318754fdf423880581e4c683034fc?auth=0127bb0028d08a9e91508a456a31d6e6#219">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/common/floating_button_page/floating_container/view.cljs
+++ b/src/status_im/common/floating_button_page/floating_container/view.cljs
@@ -1,6 +1,7 @@
 (ns status-im.common.floating-button-page.floating-container.view
   (:require
     [quo.core :as quo]
+    [quo.foundations.colors :as colors]
     [quo.theme :as quo.theme]
     [react-native.core :as rn]
     [status-im.common.floating-button-page.floating-container.style :as style]))
@@ -9,9 +10,10 @@
   [child]
   (let [theme (quo.theme/use-theme)]
     [quo/blur
-     {:blur-amount 12
-      :blur-radius 12
-      :blur-type   theme}
+     {:blur-amount        52
+      :blur-radius        20
+      :blur-type          :transparent
+      :blur-overlay-color (colors/theme-colors colors/white-70-blur colors/neutral-95-opa-70-blur theme)}
      [rn/view {:style style/blur-inner-container}
       child]]))
 

